doc: nrf5340audio: rephrase codec select section (#8519)
nrfconnect/sdk-nrf: applications/nrf5340_audio/README.rst

> Software codec requirements

Reviewer: The note starting on line 144 should be changed

===========================

> The nRF5340 Audio application must use the LC3 software codec (developed specifically for use with LE Audio) by default.

Reviewer: ```suggestion
The nRF5340 Audio application only supports the LC3 software codec (developed specifically for use with LE Audio) by default.
```

Setting up the nRF5340 Audio repositories
=========================================

The application relies on the following repositories that need to be pulled using west:

* LC3 software codec repository - The default software codec for the application, which is not open-source.

>   The application only supports the LC3 software.

Reviewer: I think it is not optimal to only write "LC3 software"

```suggestion
  The application only supports the LC3 software codec.
```

Setting up the nRF5340 Audio repositories
=========================================

The application relies on the following repositories that need to be pulled using west:

* LC3 software codec repository - The default software codec for the application, which is not open-source.

>   The application must use the LC3 software.

Reviewer: ```suggestion
  The application only supports the LC3 codec by default.
```

Author: Rephrased a bit, since we use "default" in the previous line.
